MVPDatabase.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE IF NOT EXISTS mvp_table(
+    hunt_id INT PRIMARY KEY,
+    node_number INT NOT NULL,
+    question_text VARCHAR(255),
+    answer VARCHAR (25)
+)
+
+INSERT INTO mvp_table
+VALUES(
+    1, 1, 'How many computers are there in the LoveLac room?','37' 
+
+);
+
+INSERT INTO mvp_table
```

Comment:
State which columns entering to, do not need to enter primary key as it is auto_increment
```suggestion
INSERT INTO mvp_table (node_number, question_text, answer)
```